app/controllers/posts_controller.rb

```diff
@@ -23,9 +23,10 @@ def create
     respond_to do |format|
       format.html do
         if new_post.save
-          redirect_to "/users/#{new_post.user.id}/posts/", notice: 'Success!'
+          redirect_to "/users/#{new_post.user.id}/posts/", flash: { notice: 'Success! Post has been created!' }
         else
-          render :new, alert: 'Error!'
+          render :new, flash.now[:error] = "Error, post couldn't be saved!"
+          render action: 'new'
         end
       end
```

Comment:
- [ ] When i try to see all posts am getting an error
![fog](https://user-images.githubusercontent.com/44745855/156755150-3da5cca4-e0d8-4706-8309-3acdd1a2a237.png)

Reply:
- [ ] This is happening as well when I try to see a post by ID `http://localhost:3000/users/1/posts/6` I am getting the same error